api/src/main/java/org/openmrs/module/pihcore/metadata/liberia/LiberiaLocations.java

```diff
@@ -17,6 +17,13 @@
 
     public static LocationDescriptor HARPER = new LocationDescriptor() {
         public String uuid() { return "6fa7bab5-1ca5-4b77-8c31-80c9589af952"; }
+        public String name() { return "JJ Dossen (Kouka)"; }
+        public String description() { return "JJ Dossen (Kouka)"; }
```

Comment:
We don't need a separate location for Kouka, the staging server... I would say just go ahead and change the "HARPER" variable and the "HARPER" site to "JJ_DOSSEN"... we were intending "Harper" to refer to "JJ Dossen" but we should switch to the clearer name.  It's perfectly fine for Kouka to use the "real" JJ Dossen location as well and basically be a staging server for JJ Dossen.

Reply:
I'd rather leave it as is because there are instances where we want to test a feature/module before actual activation. e.g. ED Triage/Queue. This we do in Kouka without necessarily availing the functionality in prod JJ_DOSSEN.

Reply:
Absolutely, we need a way to turn on functionality on Kouka but not on JJ Dossen, but we should be able to do this via the components enabled in pih-config-liberia-kouka vs pih-config-liberia-jjdossen... both servers can share the same location as long as their component configurations are different... this is what we do for our Haiti servers... for instance, our CI staging server is configured with the "Thomonde" location but has extra features turned on...

https://github.com/PIH/mirebalais-puppet/blob/master/mirebalais-modules/openmrs/files/config/pih-config-haiti-thomonde.json

Both the Thomonde production server and ci.pih-emr.org use this config:

https://github.com/PIH/mirebalais-puppet/blob/master/mirebalais-modules/openmrs/files/config/pih-config-haiti-thomonde.json

But then we override a custom set of components for CI:

https://github.com/PIH/mirebalais-puppet/blob/master/mirebalais-modules/openmrs/files/config/pih-config-haiti-thomonde-ci.json

https://github.com/PIH/mirebalais-puppet/blob/master/hieradata/thomonde.pih-emr.org.yaml#L4
https://github.com/PIH/mirebalais-puppet/blob/master/hieradata/ci.pih-emr.org.yaml#L3